### PR TITLE
dockerbuild: add Ubuntu Xenial support

### PR DESCRIPTION
**Pull request by lbudai**
Add ubuntu xenial support to docker build.

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

File: dbld/rules [hunk 1]
```diff
@@ -26,17 +26,17 @@ help:
...
 	@echo "already merged to your tree, which is not there by default"
 	@echo "if you work in a git clone. These files live on the 'release'"
 	@echo "branch. Make sure you have that merged, OR work from a release tarball"
 
 
-deb: deb-zesty
+deb: deb-zesty deb-xenial
 
-shell: shell-zesty
+shell: shell-zesty shell-xenial
```
Comment (bazsi) [inline dbld/rules]: I don't think these last two changes are needed, deb/shell are only meant for very simple invocation and should only use one of the targets and not all.

there's no point in executing two shells after one-another.

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by lbudai** [comment]
@bazsi: could you approve the changes?

**Pull request merged**